src/index.js

```diff
@@ -978,6 +978,30 @@ class ServerlessAppsyncPlugin {
         }
       }
 
+      if (config.sync) {
+        if (tpl.sync === true) {
+          // Use defaults
+          Properties.SyncConfig = {
+            ConflictDetection: config.sync.ConflictDetection || 'VERSION',
+          };
+        } else if (typeof tpl.sync === 'object') {
+          if (tpl.sync.lambda === true) {
+            Properties.SyncConfig = {
+              ConflictDetection: tpl.sync.ConflictDetection,
+              LambdaConflictHandlerConfig: {
+                LambdaConflictHandlerArn: tpl.sync.lambdaArn
```

Comment:
@hounded 
The way I see it, there should be at least 2 ways to configure the lambda:
- pass the funcion´s name, just like in the [lambda data source config](https://github.com/sid88in/serverless-appsync-plugin/blob/master/src/index.js#L108)
The lambda arn should then be [resolved automatically](https://github.com/sid88in/serverless-appsync-plugin/blob/master/src/index.js#L109)
- pass the lambda Arn manually, like you did here for deeper control

Also, wouldn´t we need to add 
```js
 ConflictHandler: 'LAMBDA'
```

here?

Reply:
Yup missed that one